index.html

```diff
@@ -18,14 +18,14 @@
   <div class="wrapper">
```

Comment:
- Kindly a page title, <title> in the <head> of the page adequately and briefly describes the content of the page. 
so please update it for better accessibility :+1: 
- here are good page titles:
 **Sasan portfolio**
 **Emma full-stack webdeveloper**